lib/bootloader_setup.pm

```diff
@@ -21,6 +22,10 @@ use Utils::Backends;
 
 use backend::svirt qw(SERIAL_TERMINAL_DEFAULT_DEVICE SERIAL_TERMINAL_DEFAULT_PORT SERIAL_CONSOLE_DEFAULT_DEVICE SERIAL_CONSOLE_DEFAULT_PORT);
 
+if (isotovideo::get_version() >= 39) {
```

Comment:
Not sure why CI complains about `Use of uninitialized value in numeric ge (>=)` here :(. @dzedro any idea?